packages/output/src/browser/output-contribution.ts

```diff
@@ -122,6 +127,9 @@ export class OutputContribution extends AbstractViewContribution<OutputWidget> i
     registerCommands(registry: CommandRegistry): void {
         super.registerCommands(registry);
         registry.registerCommand(OutputCommands.CLEAR__WIDGET, {
+            execute: () => this.widget.then(widget => widget.clear())
```

Comment:
`this.widget` does not look good, it will create a new widget even if widget is not opened or revealed to a user. You should use `tryGetWidget` instead.